models/linkages/R/model2netcdf.LINKAGES.R

```diff
@@ -39,9 +39,9 @@ model2netcdf.LINKAGES <- function(outdir, sitelat, sitelon, start_date = NULL, e
   
   ### Loop over years in linkages output to create separate netCDF outputs
   for (y in seq_along(years)) {
-    if (file.exists(file.path(outdir, paste(years[y], "nc", sep = ".")))) {
-      next
-    }
+    # if (file.exists(file.path(outdir, paste(years[y], "nc", sep = ".")))) {
+    #   next
+    # }
```

Comment:
Anne is this also supposed to be turned on for debugging or should it be deleted? If there's a third option would you mind just clarifying the situation in which you would un-comment it?

Reply:
👍 second what @mccabete said

Reply:
I think it was originally in here so that nc files wouldn't be remade if they already existed. But, that was a problem sometimes if I wanted to rerun an sda time step. So, I guess I'm not sure what to do. I thought about adding an overwrite flag, but that seemed like a pecan design thing more than a choice I should make.